.github/workflows/ci.yml

```diff
@@ -47,6 +47,8 @@ jobs:
           - {VERSION: "3.12", NOXSESSION: "tests-randomorder"}
           # Latest commit on the BoringSSL main branch, as of Mar 26, 2025.
           - {VERSION: "3.12", NOXSESSION: "rust,tests", OPENSSL: {TYPE: "boringssl", VERSION: "5b6b14d9966383ae5240f659fd3ed2e7e9e09a91"}}
+          # Latest tag of AWS-LC main branch, as of March 24, 2025.
+          - {VERSION: "3.12", NOXSESSION: "rust,tests", OPENSSL: {TYPE: "aws-lc", VERSION: "c80720cb125529c233cd88896586254e41284820"}}
```

Comment:
For boring we _only_ do git. For aws-lc should we think about targetting a particular release?

Reply:
Yes, I think I accidentally overwrite my comment in a merge update for this file, but I plan to tag a release to contain this particular commit. But going forward tracking a tagged release would make the most sense. I plan to add reciprocal testing from our aws-lc repository as well to ensure we are testing pyca/cryptography as well to help avoid surprise breakages.